ui/lib/core/addon/components/masked-input.hbs

```diff
@@ -75,13 +75,25 @@
     <M.Body>
       This download is
       <strong>unencrypted</strong>. Are you sure you want to download this secret data as plaintext?
+
+      <div class="has-top-margin-m">
+        <Hds::Form::Toggle::Field
+          checked={{this.stringifyDownload}}
+          {{on "change" this.toggleStringifyDownload}}
+          data-test-stringify-toggle
+          as |F|
+        >
+          <F.Label>Stringify secret value</F.Label>
+          <F.HelperText>Preserve formatting for JSON values.</F.HelperText>
```

Comment:
Minor nit, but the language here is unclear whether the current value will preserve formatting, or checking this option will preserve formatting. Maybe update to:
```suggestion
          <F.HelperText>Selecting this option preserves formatting for JSON values.</F.HelperText>
```

Reply:
I had something similar and this is the update I got from design. 🤷‍♂️

Reply:
alright then! 🤷